src/components/Person.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+export const Person = (props) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = props;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age
+        && (
```

Comment:
```suggestion
      {age && (
```
it will be better formatting